docs/changelog/0.8.1.rst

```diff
@@ -221,7 +221,7 @@ Features
   `~plasmapy.formulary.misc.Bohm_diffusion`,
   `~plasmapy.formulary.misc.magnetic_energy_density`,
   `~plasmapy.formulary.misc.magnetic_pressure`,
-  `~plasmapy.formulary.misc.mass_density`, and
+  ``plasmapy.formulary.misc.mass_density``, and
```

Comment:
Sorry this one 😆 very good catch, always a tick mark or colon somewhere

Reply:
Thanks, I found this in documentation. I hope, that now it will pass. :)